<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>870</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="titleBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>⚒️ 작업 선택 ⚒️</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="selectDogButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,130); border-color: rgb(110, 110,130,50);
border-style: solid;
border-width: 5px;border-radius: 50px </string>
   </property>
   <property name="text">
    <string>⚒️ 후아유 강아지 조명 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectTableButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,130); border-color: rgb(110, 110,130,50);
border-style: solid;
border-width: 5px;border-radius: 50px </string>
   </property>
   <property name="text">
    <string>⚒️ 후아유 테이블</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectdeskButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,130); border-color: rgb(110, 110,130,50);
border-style: solid;
border-width: 5px;border-radius: 50px </string>
   </property>
   <property name="text">
    <string>⚒️ 후아유 책상</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectChairButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,130); border-color: rgb(110, 110,130,50);
border-style: solid;
border-width: 5px;border-radius: 50px </string>
   </property>
   <property name="text">
    <string>⚒️ 후아유 의자</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectToyButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,130); border-color: rgb(110, 110,130,50);
border-style: solid;
border-width: 5px;border-radius: 50px </string>
   </property>
   <property name="text">
    <string>⚒️ 후아유 장난감</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectEtcButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>460</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,130); border-color: rgb(110, 110,130,50);
border-style: solid;
border-width: 5px;border-radius: 50px </string>
   </property>
   <property name="text">
    <string>⚒️ 기타</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkStatisticButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>770</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>작업 통계 확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
